extensions/vertx-http/deployment/src/main/resources/dev-static/css/logstream.css

```diff
@@ -0,0 +1,75 @@
+
+#logstreamFooter {
+    background: #343a40;
+    overflow-y: auto;
+    height: unset;
+}
+
+.loggerManagerIcon:hover {
+    color: #3366ac !important;        
+}
+
+.filtered {
+    display: none;
+}
+
+
+#segmentLog {
+    font-size: 0.9em;
+}
+
+#logTerminal {
+    padding-bottom: 5px;
+    line-height: 1.00;
+}
+
+#followLogButton {
+    position: fixed;
+    color: green;
+    margin-top: 5px;
+    right: 23px;
+}
+
+#informationSection {
+    position: fixed;
+    color: white;
+    font-size: 1.6em;
+    right: 23px;
+    bottom: 5px;
+}
+
+.stopLogIndicator {
+    margin-top: 10px;
+    border-top: 1px dashed #3366ac;
+}
+
+#loggerManagerNav {
+    border-bottom: 1px solid #414850;
+    /*border-top: 1px solid #414850;*/
+    max-height: 32px;
+    position: absolute;
+    width: 100%;
+    top: 0px;
+}
+
+#viewLogButton, #hideLogButton, #loggerManagerButton {
+    color: #9a9da0;
+    font-size:0.73em;
+}
+
+#hideLogButton {
+    position: absolute;
+    right: 6px;
+    bottom: -3px;
+}
+
+#resizeButton {
```

Comment:
Well you moved the contents, and moved to IDs, but now you've reserved IDs.

I was more thinking in terms of prefixing them with a container: `.log .resizeButton` for example. This allows the class names to never clash with outside stuff.

Reply:
I mostly used IDs as the javascript also get the components and do things. Happy with that ? Also makes it easier when I pull this into a extension later.

Reply:
I think you can prefix the id's with the container? `.log #resizeButton`

Reply:
Not sure if that makes sense, as Id is unique in the page.

Reply:
But it means that if another extension happens to use the same ID weird things will happen, although I guess you will probably get that anyway. Maybe just go for longer more specific ID's to reduce the chance of a name collision?

Reply:
Yea, then the JavaScript will fail. Because I plan to pull parts of this into an extension to make the logger a standalone option, I do not want to tie it to a container, as it might be on another container then, but I agree, I can make all ids/classes start with `loggerManager`, then the chance of a collision is less. I'll do that now

Reply:
Good plan.

Reply:
Done